vizro-core/docs/pages/tutorials/explore-components.md

```diff
@@ -55,23 +53,19 @@ The code below shows the steps necessary to add a box plot to the page:
         ```
 
     === "Result"
-
-        [![FirstPage1]][FirstPage1]
-
-    [FirstPage1]: ../../assets/tutorials/dashboard/dashboard21.png
-
+        [![FirstPage1]][firstpage1]
 
 ??? note "To run the dashboard in a Notebook or script"
-
     Paste the above code into a Notebook cell, run the Notebook, and evaluate it.
 
     ---
+
     If you prefer to use Python scripts to Notebooks, here's how to try out the dashboard:
 
     1. Create a new script called `app.py`.
-    2. Copy the code above into the script.
-    3. Navigate to the directory where `app.py` file is located using your terminal.
-    4. Run the script by executing the command `python app.py`.
+    1. Copy the code above into the script.
+    1. Navigate to the directory where `app.py` file is located using your terminal.
+    1. Run the script by executing the command `python app.py`.
```

Comment:
numbering

Reply:
This is OK :) It's just a markdown syntax for automated numbering rather than manual. If you look at the built text it is fine (but thanks for flagging it anyway!). I'll resolve the other places you've raised it but leave this open so you see the comment. https://vizro--873.org.readthedocs.build/en/873/pages/tutorials/explore-components/#21-add-the-first-figure

Reply:
Perfect, thanks!